<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>385</width>
     <height>244</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="selectBindPoseLayout">
      <item>
       <widget class="RKDagPoseComboBox" name="poseComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>24</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>List of Bind Poses</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadPoseButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Load Pose</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="iPoseLayout">
      <item>
       <widget class="QPushButton" name="iPoseButton">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Position in RK I-Pose</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="iPoseCheckBox">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Is Set</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="aPoseLayout">
      <item>
       <widget class="QPushButton" name="aPoseButton">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Position in RK A-Pose</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="aPoseCheckBox">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Is Set</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="tPoseLayout">
      <item>
       <widget class="QPushButton" name="tPoseButton">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Position in RK T-Pose</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="tPoseCheckBox">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Is Set</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="ePoseLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="ePoseButton">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Position in RK Export Pose</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ePoseCheckBox">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Is Set</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="bpEditorMessage">
      <property name="minimumSize">
       <size>
        <width>268</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>268</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RKDagPoseComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">rkdagposecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
